storage/rocksdb/rdb_i_s.cc

```diff
@@ -2004,6 +2076,23 @@ struct st_mysql_plugin rdb_i_s_compact_stats = {
     0,       /* flags */
 };
 
+struct st_mysql_plugin rdb_i_s_active_compact_stats = {
```

Comment:
We are going to need a documentation ticked to document this here https://www.percona.com/doc/percona-server/LATEST/myrocks/information-schema-tables.html

Reply:
Already created at https://jira.percona.com/browse/PS-7442